images/bridge2/cmd/bridge/main.go

```diff
@@ -57,6 +58,14 @@ func main() {
 			Endpoint:       getEnv("BRIDGE_TRANSLATE_URL", "http://localhost:8091/v1/transcribe"),
 			TargetLanguage: "en",
 		},
+		assistant.Agent{
+			Assistants: map[string]assistant.Client{
+				"bridge": &assistant.OpenAIClient{
+					Endpoint:      getEnv("BRIDGE_ASSISTANT_URL", "http://localhost:8092/v1/assistant"),
+					SystemMessage: assistant.DefaultSystemMessageForName("bridge"),
```

Comment:
This is fine for now, but I think we'll probably want to pull the system message from a file in one of:
- the source directory
- the session directory

What do you think?

Reply:
Yeah, I figure we'll need to spec things out more in terms of what features are expected for adding assistants. I've opened #190 with some questions around what behavior we expect for configuring additional assistants.